From 12739bd95930d3020543b3fe28c49a9e3d4bfda4 Mon Sep 17 00:00:00 2001
From: Christopher Chavez <chrischavez@gmx.us>
Date: Fri, 12 Mar 2021 07:24:18 -0600
Subject: [PATCH] Fix some implicit function declarations

Hstrdup.c: missing stdlib.h for exit()
Hstrtoul.c: missing string.h for strcmp()
---
 pTk/config/Hstrdup.c  | 1 +
 pTk/config/Hstrtoul.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/pTk/config/Hstrdup.c b/pTk/config/Hstrdup.c
index bbde6d2c..4c6b6abf 100644
--- a/pTk/config/Hstrdup.c
+++ b/pTk/config/Hstrdup.c
@@ -1,3 +1,4 @@
+#include <stdlib.h>
 #include <string.h>
 
 #define STRING "Whatever"
diff --git a/pTk/config/Hstrtoul.c b/pTk/config/Hstrtoul.c
index a5be7031..e29e4e4e 100644
--- a/pTk/config/Hstrtoul.c
+++ b/pTk/config/Hstrtoul.c
@@ -1,4 +1,5 @@
 #include <stdlib.h>
+#include <string.h>
 
 int main()
 {char *e;
